numpy/_core/src/multiarray/lowlevel_strided_loops.c.src

```diff
@@ -888,21 +891,20 @@ npy_uint64 _npy_halfbits_to_doublebits(npy_uint16 h){
 #    define _ROUND_TRIP(x) (x)
 #  elif @is_bool2@
 #    define _CONVERT_FN(x) ((npy_bool)!npy_half_iszero(x))
-#    define _ROUND_TRIP(x) npy_float_to_half((float)(!npy_half_iszero(x)))
+#    define _ROUND_TRIP(x) npy_float_to_half((!npy_half_iszero(x))
```

Comment:
Needs a `)`?

Reply:
My bad. This change was actually meant to break the build. Looking again, it makes sense: _ROUND_TRIP never gets called when the second type is bool. See:
https://github.com/numpy/numpy/blob/1b11ae0aef650345b8218bebf79af6fd0e1fe25e/numpy/_core/src/multiarray/lowlevel_strided_loops.c.src#L956-L962